services/supervisord/supervisord.go

```diff
@@ -533,6 +585,61 @@ stdout_logfile_backups = 3
 redirect_stderr = true
 {{end}}
 
+{{define "victoriametrics"}}
+[program:victoriametrics]
+priority = 7
+command =
+	/usr/sbin/victoriametrics
+		--promscrape.config=/etc/victoriametrics-promscrape.yml
+		--retentionPeriod={{ .DataRetentionMonths }}
+		--storageDataPath=/srv/victoriametrics/data
+		--httpListenAddr=127.0.0.1:8428
+		--search.disableCache={{.VMDBCacheDisable}}
+user = pmm
+autorestart = {{ .IsVMEnabled }}
+autostart = {{ .IsVMEnabled }}
+startretries = 10
+startsecs = 1
+stopsignal = INT
+stopwaitsecs = 300
+stdout_logfile = /srv/logs/victoriametrics.log
+stdout_logfile_maxbytes = 10MB
+stdout_logfile_backups = 3
+redirect_stderr = true
+{{end}}
+
+{{define "vmalert"}}
+[program:vmalert]
+priority = 7
+command =
+	/usr/sbin/vmalert
+        --notifier.url="http://127.0.0.1:9093/alertmanager{{- if .AlertmanagerURL }},{{ .AlertmanagerURL }}{{- end }}"
```

Comment:
Please pass fully prepared parameters to template to avoid `if` and especially `{{-`